pages/main_page.py

```diff
@@ -0,0 +1,28 @@
+from pages.base_page import BasePage
+from selenium.webdriver.common.by import By
+
+
+button_sign_in_selector = (By.CSS_SELECTOR, '[data-ga-stats-name="auth_block_login"]')
+icon_selector = (By.CLASS_NAME, '_1tlqpaw1 _15r2d5o1')
```

Comment:
Ouch, this generated one time selectors, if you run your code next time it may not work. Try to change